packages/common/dist/loader.js

```diff
@@ -9,10 +9,10 @@ export class Loader {
     // Returns true if ENgrid should reload (that means the current ENgrid is not the right one)
     // Returns false if ENgrid should not reload (that means the current ENgrid is the right one)
     reload() {
-        var _a, _b, _c, _d, _e;
+        var _a, _b, _c, _d;
         const isLoaded = ENGrid.getBodyData("loaded");
         let assets = this.getOption("assets");
-        if (isLoaded || !assets) {
```

Comment:
@fernanDOTdo @bryancasler In order for the loader to always check the `engridcss` and `engridjs` query parameters, I removed this guard.

The only behavior difference I can see is that if the `assets` parameter is not provided, the default condition which loads the exeternal JS and CSS files will now run. Whereas before it would not run if the `assets` parameter was not provided.

Is this desirable/undesirable? If it's important for that default condition to only run when `assets` is set, I can revise the logic.

Reply:
After thinking about this more, I added a condition before the switch statement to only run that if `assets` is provided. I think that preserves the existing behavior while always checking the `engridcss` and `engridjs` parameters.